### ISSUE TITLE
Not able to build the jar for Stdout Option for the collector

### ISSUE THREAD

**Issue by tgampiyush** [snowplow/snowplow#4280]
**Project**: Scala Stream Collector - stdout

**Version**: 1.0.0

**Expected behavior**: The jar should be created with name - snowplow-stream-collector-stdout-{$versionn-no}.jar 

**Actual behavior**: The project just clean and compile and no jar is created

**Steps to reproduce**:
 sbt "project stdout" assembly

 1. Just try to create an assembly for scala stream collector module where the output set to stdout  - 
using the command - 
 sbt "project stdout" assembly

**PR already create to resolve it -**
https://github.com/snowplow/snowplow/pull/4278

**Comment by chuwy** [comment]
Thanks @tgampiyush! Hey @benjben or @dilyand am I right that this is a regrerssion from R118?

**Comment by chuwy** [comment]
Or actually from R117 https://github.com/snowplow/snowplow/blob/master/CHANGELOG#L119

**Comment by benjben** [comment]
@chuwy there is no `scala-stream-collector-stdout` on our [Bintray](https://bintray.com/snowplow/registry), so I'm not sure that it has ever been published somewhere.

Already in R114 before `sbt-native-packager` was introduced we were [not publishing it](https://github.com/snowplow/snowplow/blob/r113-filitosa/.travis/release_scala_stream_collector.yml#L26-L29).

**Comment by chuwy** [comment]
Yup, I spotted [this ticket](https://github.com/snowplow/snowplow-docker/issues/54) today morning. Not sure what's going on with #4278, but at least this is clearly not a regression.
